apps/api/src/app/inbox/usecases/snooze-notification/snooze-notification.usecase.ts

```diff
@@ -74,6 +79,19 @@ export class SnoozeNotification {
           this.logger.error({ err: error }, 'Failed to create execution details');
         });
 
+      const subscriber = await this.getSubscriber.execute({
```

Comment:
Let's avoid fetching the subscriber from the database. Instead, you can rely on the notification that is already been fetched the notification should have a notification._subscriber ID on it.